### PR TITLE
New test case for unions: unions with complex fields [20586]

### PR DESCRIPTION
**Pull request by richiware**

### CODE REVIEW

File: IDL/constants.idl [hunk 1]
```diff
@@ -1,94 +1,94 @@
...
+    short array_literal_const_inner_const_helper[inner_const_helper];
+
+    short array_moduled1_literal_const[const_module1::const_moduled];
+    short array_moduled2_literal_const[const_module2::const_moduled];
+
+    short array_literal_const_alias_const[alias_const];
+    short array_moduled1_literal_alias_const_moduled[const_module1::alias_const_moduled];
+    short array_moduled2_literal_alias_const_moduled[const_module2::alias_const_moduled];
+};
+
```
Comment (adriancampo) [inline IDL/constants.idl]:
```suggestion
};
```
Removed extra blank line

File: IDL/arrays.idl [hunk 2]
```diff
@@ -1,535 +1,535 @@
...
+struct BoundedSmallArrays
+{
+    short var_array_small[1];
+};
+
+struct BoundedBigArrays
+{
+    short var_array_big[41925];
+};
+
```
Comment (adriancampo) [inline IDL/arrays.idl:535]:
```suggestion
};
```
Removed extra blank line

File: IDL/sequences.idl [hunk 3]
```diff
@@ -1,144 +1,144 @@
...
+    sequence<short, 1> var_sequence_small;
+    sequence<string, 5> var_unbounded_string_small_bounded_sequence;
+};
+
+struct BoundedBigSequences
+{
+    sequence<short, 41925> var_sequence_big;
+    sequence<string, 256> var_unbounded_string_large_bounded_sequence;
+};
+
```
Comment (adriancampo) [inline IDL/sequences.idl:144]:
```suggestion
};
```
Removed extra blank line

File: IDL/strings.idl [hunk 4]
```diff
@@ -1,29 +1,29 @@
...
+
+struct LargeStringStruct
+{
+    string<41925> var_large_string;
+};
+
+struct LargeWStringStruct
+{
+    wstring<41925> var_large_wstring;
 };
```
Comment (adriancampo) [inline IDL/strings.idl]:
```suggestion
};

```
Added extra blank line

File: IDL/arrays.idl (hunk 2 above)
Comment (adriancampo) [inline IDL/arrays.idl]:
```suggestion
};
```
Removed extra blank line

File: IDL/sequences.idl (hunk 3 above)
Comment (adriancampo) [inline IDL/sequences.idl]:
```suggestion
};
```
Removed extra blank line

File: IDL/constants.idl (hunk 1 above)
Comment (richiware) [reply IDL/constants.idl]: Done in e431e0e

File: IDL/strings.idl (hunk 4 above)
Comment (richiware) [reply IDL/strings.idl]: Done in e431e0e

File: IDL/arrays.idl (hunk 2 above)
Comment (richiware) [reply IDL/arrays.idl]: Done in e431e0e

File: IDL/sequences.idl (hunk 3 above)
Comment (richiware) [reply IDL/sequences.idl]: Done in e431e0e

**Review by adriancampo — APPROVED** [review]
LGTM

**Pull request merged**